src/clusterfuzz/_internal/issue_management/request_cache.py

```diff
@@ -15,19 +15,19 @@
 
```

Comment:
Actually, can we move this back to appengine.libs ? This is very appengine specific and should belong there.